LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Temporal Logics @ Sapienza
+Copyright (c) 2019 Logics for AI @ Sapienza
```

Comment:
shouldn't that be `Marco Favorito, Francesco Fuggitti 2019`?

Reply:
I think the owner is Logics for AI, we are the authors